src/equations/ideal_glm_mhd_multicomponent_1d.jl

```diff
@@ -86,6 +86,39 @@ function initial_condition_convergence_test(x, t, equations::IdealGlmMhdMulticom
 end
 
 
+"""
+    initial_condition_weak_blast_wave(x, t, equations::IdealGlmMhdMulticomponentEquations1D)
+A weak blast wave adapted from
```

Comment:
```suggestion
    initial_condition_weak_blast_wave(x, t, equations::IdealGlmMhdMulticomponentEquations1D)

A weak blast wave adapted from
```
The docstring does not render correctly otherwise